apps/state/lib/state/route_pattern.ex

```diff
@@ -71,6 +71,8 @@ defmodule State.RoutePattern do
         _ -> []
       end
 
+    opts = Keyword.put(opts, :canonical?, false)
```

Comment:
I did look at that before going the route I did. The main reason I didn't is because this is used in several places that aren't specifying `canonical?` and so are getting the default true for example [routes_by_family_stops](https://github.com/mbta/api/blob/431c5738570f7d5f814d28c503db830d480e6c48/apps/state/lib/state/routes_patterns_at_stop.ex#L31). Admittedly though, it's not even clear to me there that default true is the right thing to do.

I actually do think there should be a follow-up on this. This isn't the first ticket that's led me to `canonical?` and consequently [stops_on_route_by_shape](https://github.com/mbta/api/blob/431c5738570f7d5f814d28c503db830d480e6c48/apps/state/lib/state/helpers.ex#L44) (the return value there ends up in the `canonical` field for `RoutePatternsAtStop` and `StopsOnRoute`), but I think it's a deep dive to figure out all the ways it's being used and what the correct behaviors are.